dictionaries/html/src/html.txt

```diff
@@ -1,24 +1,28 @@
+__utma
+__utmb
+__utmc
+__utmk
+__utmt
+__utmv
+__utmx
+__utmz
 a
 aacute
-Aacute
```

Comment:
These are actually different from the lower case, since they're the upper case version of the entities. Alternately, you could drop these, and merge the html-entites dictionary on build